### PR TITLE
add .gitattributes file

### PR DESCRIPTION
**Pull request by gggeek**
## Description
Implement issue #104 : add .gitattributes file to shrink the 'dist' version of the package

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] I updated the documentation (see [here](https://github.com/kevinpapst/AdminLTEBundle/tree/master/Resources/docs))
- [x] I agree that this code is used in AdminLTEBundle and will be published under the [MIT license](https://github.com/kevinpapst/AdminLTEBundle/blob/master/LICENSE)

### CODE REVIEW

**Comment by gggeek** [comment]
Note: I am still a noobie with nodejs/yarn/webpack.
I am not sure if the files _package.json_ and _yarn.lock_ could also be removed from the 'dist' package, as the building of the AdminLTEBundle bundle assets is done before a release is made and pushed to github, or if they are necessary, as those assets are built when yarn/webpack are run in the top-level project...

**Comment by kevinpapst** [comment]
yarn.lock can be removed.
Frontend tooling is not my business as well, so I am not entirely sure if package.json might used by any tool. I'd say no, but I am not entirely sure, so lets keep it for now.

**Comment by gggeek** [comment]
updated

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.github export-ignore
+Tests export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.php_cs export-ignore
+.travis.yml export-ignore
+phpstan.neon export-ignore
+phpunit.xml.dist export-ignore
+yarn.lock
```
Comment (kevinpapst) [inline .gitattributes]: the `export-ignore` is missing here?!

Comment (gggeek) [reply .gitattributes]: doh

**Comment by kevinpapst** [comment]
What about https://github.com/kevinpapst/AdminLTEBundle/tree/master/Resources/docs ?

**Comment by gggeek** [comment]
I think those should be included, as they are useful to the developer who wants to use this bundle

**Comment by kevinpapst** [comment]
thanks @gggeek 👍

**Pull request merged**